src/demo-app/select/select-demo.html

```diff
@@ -144,6 +144,59 @@
     </mat-card-content>
   </mat-card>
 
+  <mat-card class=".demo-appearance-comparison">
```

Comment:
Can you add a button to this demo that allows me to toggle all of the selects to either have a value or be cleared? It's tough to see the animation when manually selecting the value

Reply:
Sure, will do!